docs/source/general/guides/drift_guide.rst

```diff
@@ -11,6 +11,10 @@ This guide will help you understand what drift is and how you can detect in usin
 * `Which Types of Drift Are There? <#which-types-of-drift-are-there>`__
 * `How Do You Detect Drift? <#how-do-you-detect-drift>`__
 * `How Can I Use Deepchecks to Detect Drift? <#how-can-i-use-deepchecks-to-detect-drift>`__
+
+    * `Tabular Data <#tabular-data>`__
```

Comment:
bullets should be of a different kind (and I think that the indentation has to be of exactly by 2)

Reply:
But you were right about double space only. Both work but this looks better